src/user/proposals/AdminProposalDashboard/DashboardContent/DashboardContent.js

```diff
@@ -2,12 +2,143 @@ import React, { Component } from "react";
 import { Button, Form, Col, Image } from "react-bootstrap";
 import "./DashboardContent.scss";
 import userIcon2 from "../../../../images/userIcon2.jpg";
+import { Link } from "react-router-dom";
 
 class DashboardContent extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      token:
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI1ZWVlZWY5OTdhZTUyZDJlN2NjNmFhZjEiLCJpYXQiOjE1OTI3MTcyMTB9.ZmIn4pIuwWygI_YJrBo8CIhpIhmwZmAE5wsoDhVcpE8",
+      userId: "5eeeef997ae52d2e7cc6aaf1",
+      proposals: [],
+      displayItems: [],
+    };
   }
+
+  componentDidMount() {
+    this.getProposalsByUserId();
+    this.getAllProposals();
+  }
+
+  getAllProposals() {
```

Comment:
Make these as actions in actions folder, and dispatch from there to update your state.